sdk/src/state_machine/stf.rs

```diff
@@ -56,18 +83,11 @@ pub trait StateTransitionFunction {
     fn begin_slot(&mut self);
 
     /// Apply a batch of transactions to the rollup, slashing the sequencer who proposed the batch on failure
```

Comment:
Let's change `batch` to `blob` in the doc.

Reply:
Fixed by @citizen-stig